src/lib/evas/canvas/meson.build

```
@@ -1,7 +1,49 @@
+pub_eo_files_optional_ector_ = [
```

For temporary/internal variables, I prefer to _prefix_ with `_` instead of suffixing, since it is more unlikely that someone should be using a variable that starts with `_`.

```suggestion
_pub_eo_files_optional_ector = [
```